src/renderers/common/Renderer.js

```diff
@@ -2033,29 +2033,6 @@ class Renderer {
 
 	}
 
-	/**
-	 * Ensures the renderer is XR compatible.
-	 *
-	 * @async
-	 * @return {Promise} A Promise that resolve when the renderer is XR compatible.
-	 */
-	async makeXRCompatible() {
-
-		await this.backend.makeXRCompatible();
```

Comment:
Wouldn't it be interesting to have these functions in the `Renderer` for backward compatibility? Maybe this simplifies it a bit when WebXR is for WebGPU.

Reply:
I wanted to avoid these methods to be in the public renderer interface since they are meant to be internal. It should be sufficient to have the API defined in the `Backend` class since the XR manager now operates directly on the backend.

Reply:
I also prepare a follow-up PR to slightly reorganize the XR render target handling. The WebXR Layers configuration isn't trivial, tbh.